dev-client/src/components/links/InternalLink.tsx

```diff
@@ -17,15 +17,27 @@
 
 import {Link} from 'native-base';
 import {InterfaceLinkProps} from 'native-base/lib/typescript/components/primitives/Link/types';
+import {useCallback} from 'react';
+import {Linking} from 'react-native';
 
 type InternalLinkProps = {
   label: string;
   onPress?: InterfaceLinkProps['onPress'];
+  url?: string;
```

Comment:
I added direct URL support to InternalLink since it was decided that we won't be implementing embedded web views for this release.

Reply:
i think that makes sense either way? it still seems nice to encapsulate that logic!